tests/admin_custom/apps.py

```diff
@@ -0,0 +1,8 @@
+from __future__ import unicode_literals
```

Comment:
I don't really know: for me they are different things, so they need different tests.
But not really a problem merging them.